flipt-client-browser/src/index.ts

```diff
@@ -100,7 +100,8 @@ export class FliptEvaluationClient {
 
     const data = await resp.json();
 
-    const engine = new Engine(namespace, data);
+    const engine = new Engine(namespace);
+    engine.snapshot(data);
```

Comment:
we'll have to do the same thing in the node SDK now as well I think

Reply:
I missed that. Ty!